coriolisclient/cli/utils.py

```diff
@@ -185,3 +188,34 @@ def get_option_value_from_args(args, option_name, error_on_no_value=True):
             "No '%s[-file]' parameter was provided." % option_arg_name)
 
     return value
+
+
+def compose_user_scripts(global_scripts, instance_scripts):
+    ret = {
+        "global":{},
+        "instances": {}
+    }
+    global_scripts = global_scripts or []
+    instance_scripts = instance_scripts or []
+    for glb in global_scripts:
+        split = glb.split("=", 1)
+        if len(split) != 2:
```

Comment:
I think we should also raise a `ValueError` here if any arg is improperly formatted instead of silently skipping over it it.